CHANGES/936.contrib.rst

```diff
@@ -0,0 +1,2 @@
+Added test coverage for the :ref:`and <python:and>`, :ref:`or <python:or>`, :ref:`sub <python:sub>`, and :ref:`xor <python:xor>` operators in the :file:`multidict/_multidict_base.py` module. It also covers  
```

Comment:
Looks like the labels only exist for and/or. Let's link something else for the rest:
```suggestion
Added test coverage for the :ref:`and <python:and>`, :ref:`or <python:or>`, :py:obj:`sub <python:object.__sub__>`, and :py:obj:`xor <python:object.xor>` operators in the :file:`multidict/_multidict_base.py` module. It also covers  
```